src/components/BookDetails.js

```diff
@@ -17,27 +22,31 @@ const BookDetails = ({
   return (
     <section className="book-element--container">
       <div className="book-info">
-        <p>{category}</p>
-        <p>{title}</p>
-        <p>{author}</p>
+        <p className="category">{category}</p>
```

Comment:
- [ ] Please use the correct foont for the `category` part of the book details, the correct font is `font-family: Montserrat;` and not `Roboto`